src/Core/AspectKernel.php

```diff
@@ -65,6 +73,13 @@
     protected $container = null;
 
     /**
+     * Cache path resolver instance
+     *
+     * @var null|CachePathResolver
+     */
+    protected $cachePathResolver = null;
```

Comment:
See my previous comment about the usage of container for storing instances. DI is the best way to decouple the source code.